### PR TITLE
[ch-18567] Get tool_path_cost to reduce redundant motion

### PR DESCRIPTION
**Pull request by CUN-bjy**
# PR Description
- custom editing of `ToolGoalPose` cost function in order to get tool_path_cost from trajectory.

## Shortcut Link
- [ch-18567](https://app.shortcut.com/makinarocks/story/18567/trajectory-optimization-for-reducing-redundant-motion)

### CODE REVIEW

**Comment by CUN-bjy** [comment]
ping @yu2jeong @yurirocha15 @leejeyeol

File: stomp_core/src/stomp.cpp [hunk 1]
```diff
@@ -248,11 +248,12 @@ bool Stomp::solve(const Eigen::MatrixXd& initial_parameters,
     return false;
   }
 
+  ROS_WARN_STREAM("Start Interation");
   parameters_valid_prev_ = parameters_valid_;
   while(current_iteration_ <= config_.num_iterations && runSingleIteration())
   {
 
-    ROS_DEBUG("STOMP completed iteration %i with cost %f",current_iteration_,current_lowest_cost_);
+    ROS_WARN("STOMP completed iteration %i with cost %f",current_iteration_,current_lowest_cost_);
```
Comment (yurirocha15) [inline stomp_core/src/stomp.cpp:256]: Shouldn't we change all those warns to infos?

**Review by yu2jeong — APPROVED** [review]
LGTM

File: stomp_core/src/stomp.cpp (hunk 1 above)
Comment (CUN-bjy) [reply stomp_core/src/stomp.cpp]: fixed in e709ac1

**Review by yurirocha15 — APPROVED** [review]
LGTM

**Pull request merged**